src/main/java/io/github/thebusybiscuit/slimefun4/utils/BlockUtils.java

```diff
@@ -15,32 +16,26 @@ public static boolean hasInventory(Block block) {
             case DROPPER:
             case HOPPER:
             case BREWING_STAND:
-            case SHULKER_BOX:
-            case WHITE_SHULKER_BOX:
-            case ORANGE_SHULKER_BOX:
-            case MAGENTA_SHULKER_BOX:
-            case LIGHT_BLUE_SHULKER_BOX:
-            case YELLOW_SHULKER_BOX:
-            case LIME_SHULKER_BOX:
-            case PINK_SHULKER_BOX:
-            case GRAY_SHULKER_BOX:
-            case LIGHT_GRAY_SHULKER_BOX:
-            case CYAN_SHULKER_BOX:
-            case PURPLE_SHULKER_BOX:
-            case BLUE_SHULKER_BOX:
-            case BROWN_SHULKER_BOX:
-            case GREEN_SHULKER_BOX:
-            case RED_SHULKER_BOX:
-            case BLACK_SHULKER_BOX:
             case ENDER_CHEST:
             case BARREL:
             case BLAST_FURNACE:
             case LECTERN:
             case SMOKER:
                 return true;
             default:
-                return false;
+                return isShulkerBox(block);
         }
     }
 
+    public static boolean isShulkerBox(Block block) {
+        Material type = block.getType();
+        return type == Material.SHULKER_BOX || type == Material.WHITE_SHULKER_BOX || type == Material.ORANGE_SHULKER_BOX
```

Comment:
I'd just do name().endsWith("SHULKER_BOX") saves a lot of code

Reply:
Should I remove the method?

Reply:
I agree, this all can be reduced to the simple .endsWith statement which would be much simpler.